EIPS/eip-2003.md

```diff
@@ -0,0 +1,108 @@
+---
+eip: 2003
+title: EVMC modules for implementations of precompiled contracts
+author: Paweł Bylica (@chfast)
+discussions-to: HERE
```

Comment:
Should this be a magicians link or an issue on the EVMC repo? I think https://github.com/ethereum/evmc/issues/259 could be a good discussion URL.

Reply:
Good idea.